scripts/run.py

```diff
@@ -934,15 +934,15 @@ def _pytest_cmd(test_name, ip, port, on_hadoop, hadoop_namenode):
         return cmd
 
     def _javascript_cmd(self, test_name, ip, port):
-        # return ["phantomjs", test_name]
-        if g_perf:
-            return ["phantomjs", test_name, "--host", ip + ":" + str(port), "--timeout", str(g_phantomjs_to),
-                    "--packs", g_phantomjs_packs, "--perf", g_date, str(g_build_id), g_git_hash, g_git_branch,
-                   str(g_ncpu), g_os, g_job_name, g_output_dir, "--excludeFlows", self.exclude_flows]
-        
-        else:
-            return ["phantomjs", test_name, "--host", ip + ":" + str(port), "--timeout", str(g_phantomjs_to),
-                    "--packs", g_phantomjs_packs, "--excludeFlows", self.exclude_flows]
+        return ["phantomjs", test_name]
```

Comment:
@mmalohlava the new UI Tests are here https://github.com/h2oai/drift/tree/master/tests/ui-tests

I like to run them with a live browser so that I can visually inspect the state after a test,  which is harder to do with phantomjs. The downside to UI tests alone is that they rely on the developer to run them manually.  It would be nice to eventually have both UI tests and automated phantomjs headless browser tests.